pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/kit/KitStatusChangeRoute.java

```diff
@@ -18,13 +18,16 @@
 public abstract class KitStatusChangeRoute extends RequestHandler {
 
     protected NotificationUtil notificationUtil;
-    protected KitPayload kitPayload;
```

Comment:
We should never track request/response values in instance variables in a route this way because our routes are singletons, and different threads can overwrite these values when multiple requests are running simultaneously.